zh/guide/ch_invest.docbook

```diff
@@ -343,165 +266,136 @@
       </para>
     </sect2>
 
     <sect2 id="invest_creating2">
-      <title>Creating Investment Accounts Manually</title>
+      <title>手动创建投资科目</title>
 
-      <para>If you want to set up your own investment accounts hierarchy, you may of course do so. Investments
-        usually have a number of associated accounts that need to be created: an asset account to
-        track the investment itself; an income account to track dividend transactions; and expense
-        accounts to track investment fees and commissions.
+      <para>当然也可以自行建立投资科目层次。处理投资事宜常常需要创建几个相关联的科目：用于跟踪投资资金自身的资产科目、用于跟踪分红明细的收入科目，以及用于记录投资手续费的费用科目。
       </para>
 
-      <para>In a typical account structure, security accounts are sub accounts of an asset account representing
-        an account at a brokerage firm. The brokerage account would be denominated in your local
-        currency and it would include sub accounts for each security that you trade there.
+      <para>在典型的科目结构中，证券科目是资产科目的子科目，代表在某一经纪公司处开立的一个账户。经纪账户以本位货币计价，其下包括与每种在该经纪公司处交易的证券相对应的子科目。
       </para>
 
-      <para>Related purchases, sales, income and expense accounts should also be in the same currency as the
-        brokerage account.
+      <para>相关的买入、卖出、收入和费用科目的货币也应当与经纪科目一致。
       </para>
 
-      <para>The security sub accounts would each be configured to contain units of a single security selected
-        from the master (user defined) security list and they are expected to use the same currency
-        as the brokerage account.
+      <para>每个证券子科目应当设为从（用户定义的）主证券列表中选择的单种证券单位，同时使用与经纪科目一致的货币。
       </para>
 
-      <para>Security prices are kept in the <guilabel>Price Database</guilabel> of &app; (see
-        <xref linkend="invest-stockprice1" />). This contains prices for individual securities (not
-        security accounts). All prices for an individual security are in a single currency. If a
-        security is traded in multiple currencies, then a separate security and separate accounts
-        should be set up for each currency.
+      <para>证券价格记录在&app;的<guilabel>价格数据库</guilabel>中（参见<xref linkend="invest-stockprice1" />）。其中包含每一种证券单位（而非证券科目）的价格。单种证券的所有价格都由同一种货币记录。如果某种证券使用多种货币交易，则应当为每种货币建立各自独立的证券单位和证券科目。
       </para>
     </sect2>
 
     <sect2 id="invest_custom2">
-      <title>Custom Accounts Example</title>
+      <title>自建科目示例</title>
 
-      <para>The following is a somewhat more complicated example of setting up &app; to track your investments,
-        which has the advantage that it groups each different investment under the brokerage that
-        deals with the investments. This way it is easier to compare the statements you get from
-        your brokerage with the accounts you have in &app; and spot where &app; differs from the
-        statement.
+      <para>下面是一个稍显复杂的示例，展示了怎样设置用于跟踪投资的&app;的科目结构。其优势在于将每种不同的投资对象分组在处理投资交易的经纪商之下。这样更便于比对经纪商给出的账单与&app;中已有的科目交易记录，并定位&app;与账单间的差异。
+      </para>
+        
+      <simpara>
```

Comment:
This unintended insertion is breaking the check.
`ninja zh-guide-check` returns
`index.docbook:305: element simpara: validity error : Element screen is not declared in simpara list of possible children`

Because we are using `<xi:include href="…" />` check returns an unusable location.
To find the right  location I have run
```
grep -inr '<screen>' *.docbook
grep -inr '<simpara' ch_invest.docbook
```